runner/src/run.rs

```diff
@@ -85,6 +99,11 @@ fn launch_qemu(args: &RunArgs, miralis: PathBuf, firmware: PathBuf) -> ExitCode
     if let Some(cpu) = &cfg.qemu.cpu {
         qemu_cmd.arg("-cpu").arg(cpu);
     }
+
+    if is_qemu_version_6() {
+        qemu_cmd.arg("-cpu").arg("-cpu rv64,h=true");
```

Comment:
I would also add a `log::debug!` here to tell the user we enable the `h` extension. This might cause some unexpected and hard to debug behavior (say if in the future we add some conflicting CPU option, or if the user sets `h=false` in its config which is already possible today).